pkg/cloud/services/autoscaling/autoscalinggroup_test.go

```diff
@@ -18,20 +18,25 @@ package asg
 
```

Comment:
If [BuildTags](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/9a86ec7d19650f754cb20882f9b5baa3df802529/pkg/cloud/services/autoscaling/autoscalinggroup.go#L394) func is not used anywhere, can we delete it?

Reply:
@sedefsavas can you comment here if this is ok?

Reply:
Yes, let's cleanup unused methods.